docs/modules/ROOT/pages/references/functional-requirements.adoc

```diff
@@ -0,0 +1,91 @@
+= Functional Requirements
+
+This page discusses functional requirements.
+It currently mainly targets the Control API and the {product} Portal.
+
+== Feature: Authentication
+
+> As a xref:references/glossary.adoc#_user[User] +
+> I want to only login once to the {product} ecosystem +
+> So that I can use strong authentication methods and still get a good user experience
+
+*Scenarios*
+
+* Login
+* Logout
+
+== Feature: Manage Organizations
+
+> As a xref:references/glossary.adoc#_user[User] +
+> I want to manage organizations +
+> So that I can group resources under it
+
+*Scenarios*
+
+* Create Organization
+* List Organizations
+* Edit Organization
```

Comment:
What's the scope of "editing"? Does editing include editing teams? Or just metadata like display name?
I think we should be a little bit more specific here as it can be easily misunderstood.